factory/factory.go

```diff
@@ -46,14 +49,10 @@ func InitConfigFactory(f string) error {
 			SmfConfig.Configuration.KafkaInfo.EnableKafka = &enableKafka
 		}
 
-		roc := os.Getenv("MANAGED_BY_CONFIG_POD")
-		if roc == "true" {
-			gClient := client.ConnectToConfigServer(SmfConfig.Configuration.WebuiUri)
-			commChannel := gClient.PublishOnConfigChange(false)
-			go SmfConfig.updateConfig(commChannel)
+		if os.Getenv("MANAGED_BY_CONFIG_POD") == "true" {
+			logger.CfgLog.Infoln("MANAGED_BY_CONFIG_POD is true")
 		}
```

Comment:
Why do you need this `if` statement here given that you are already adding a similar `if` statement in `service/init.go` (same log message "MANAGED_BY_CONFIG_POD is true")?

Reply:
Removed.